src/compressed_tensors/utils/match.py

```diff
@@ -72,8 +77,8 @@ def match_named_modules(
 
 def match_named_parameters(
     model: torch.nn.Module,
-    targets: Iterable[str],
-    ignore: Iterable[str] = tuple(),
+    targets: Iterable[str] | None = None,
+    ignore: Iterable[str] | None = None,
```

Comment:
pipe type hinting is only supported in python3>=3.10, please use `Union` for now